src/game/AI/ScriptDevAI/scripts/world/scourge_invasion.cpp

```diff
@@ -37,8 +37,8 @@ Unit* SelectRandomFlameshockerSpawnTarget(Creature* unit, Unit* except, float ra
 {
     std::list<Unit*> targets;
 
-    MaNGOS::AnyUnfriendlyUnitInObjectRangeCheck u_check(unit, radius);
-    MaNGOS::UnitListSearcher<MaNGOS::AnyUnfriendlyUnitInObjectRangeCheck> searcher(targets, u_check);
+    MaNGOS::AnyUnitInObjectRangeCheck u_check(unit, radius);
```

Comment:
UnfriendlyUnit has an LOS check, which we don't want because the summons are supposed to occur city-wide